### DISCUSSION TITLE
Last appeal

### DISCUSSION THREAD

**Discussion by ghost** [discordjs/discord.js#10409] [Discord Server Ban]
I didn’t even use another account is only two and I sometimes left and rejoin like seriously how am i switching between accounts? Is just I delete that account and I’m now using this account right now! Is still the same account I use!!

**Comment by ghost** [comment]
And also that old account I had I deleted cuz it had so much hacked ans wrongs so I can join with this account and please make sense with it! And also I did a stupid thing and I’Ll promise never left and rejoin again I’ll stay on server

**Comment by almostSouji** [comment]
Please refer to https://github.com/discordjs/discord.js/discussions/10406#discussioncomment-10135049
